app/controllers/api/admin/users_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::Admin::UsersController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    authorize(current_user)
+    user = User.find(params[:id])
```

Comment:
Why do you authorize for `current_user`?

Reply:
We had a silly idea that it was needed for our restriction. Turned out that and the policies were redundant. Fixed